frame/assets/src/types.rs

```diff
@@ -260,9 +264,15 @@ where
 		// make sure we don't divide by zero
 		// ensure!(!min_balance.is_zero(), ConversionError::MinBalanceZero);
 		let balance = CON::convert(balance);
-		// balance * asset.min_balance / min_balance(1_000_000)
+
+		let para_fee_rate = ParaFeeRate::<T, I>::get().ok_or(ConversionError::AssetMissing)?;
+
+		// balance * para_fee_rate / (system_token_weight * correction_para_fee_rate)
 		// ToDo: Divisor should be changed based on the decimals
-		Ok(FixedU128::saturating_from_rational(asset.min_balance, 1_000_000u128)
-			.saturating_mul_int(balance))
+		Ok(FixedU128::saturating_from_rational(
+			para_fee_rate,
+			asset.system_token_weight * CORRECTION_PARA_FEE_RATE,
```

Comment:
Can't we just mul '1' if `ParaFeeRate::<T,I>::get() == 1`?

Reply:
To achieve ParaFeeRate values such as 1.2 or 0.8, the default value was set to 1_000. Is it okay to leave it as is?

Reply:
To achieve ParaFeeRate values such as 1.2 or 0.8, the default value was set to 1_000. Is it okay to leave it as is?

Reply:
got it!